objects/tokenList.py

```diff
@@ -92,93 +89,11 @@ async def getUserIDFromToken(token_id: str) -> Optional[int]:
     """
     token = await osuToken.get_token(token_id)
     if token is None:
-        return
+        return None
 
     return token["user_id"]
 
 
-@overload
-async def getTokenFromUserID(
-    userID: int,
-    _all: Literal[False] = False,
-) -> Optional[osuToken.Token]: ...
-
-
-@overload
-async def getTokenFromUserID(
-    userID: int,
-    _all: Literal[True] = ...,
-) -> list[osuToken.Token]: ...
-
-
-async def getTokenFromUserID(
```

Comment:
Wow a _lot_ of places were calling into these, which were fetching all active sessions.

Now they're calling into the osuToken variants of the same function, which use the indices we've setup in redis.

There may be _some_ level of behaviour change here for users with multiple clients online, but I think it's okay.